src/plugins/tooltip/directives/directive.ts

```diff
@@ -11,41 +11,80 @@ import {
   DEBOUNCE_CLOSE,
   DEFAULT_POPPER_OPTIONS,
 } from '@/plugins/tooltip/shared';
+import type { NamedDirective } from '@/types/named-directive';
 
 const storageKey = Symbol('Tooltip directive instance');
+const tooltipAnchor = Symbol('The current tooltip anchor');
 
 type TooltipEvent = {
   [key: string]: EventListener;
 }
 
-interface TooltipElement extends HTMLElement {
+interface AnchorElement extends HTMLElement {
   [storageKey]: {
     isHidden: boolean;
     popper: Instance;
     events: TooltipEvent[];
+    content: string;
   };
 }
 
+interface Tooltip extends HTMLDivElement {
```

Comment:
todo: Please make the necessary updates:

`type Tooltip = HTMLDivElement & {
  [tooltipAnchor]?: HTMLElement;
}`